tests/integrationv2/configuration.py

```diff
@@ -341,9 +341,7 @@
                     " *.insect.hexapod when RSA ciphers are higher priority than ECDSA in sever preferences.",
         server_certs=[ SNI_CERTS["wildcard_insect"], SNI_CERTS["alligator"], SNI_CERTS["underwing"] ],
         client_sni="underwing.insect.hexapod",
-        # AES128-GCM-SHA256 is prioritized about ECDHE-ECDSA-AES128-SHA in
-        # the "test_all" server cipher preferences
-        client_ciphers=[Ciphers.AES128_GCM_SHA256, Ciphers.ECDHE_ECDSA_AES128_SHA],
+        client_ciphers=[Ciphers.AES128_GCM_SHA256],
```

Comment:
Based on this test's description (also nit: "higher priority than ECDSA in se**r**ver"), shouldn't we have an ECDSA cipher here so that we can verify it takes priority over RSA?

Reply:
Added ECDSA and RSA ciphers here.